packages/@o3r/core/schematics/sdk/index.ts

```diff
@@ -19,7 +19,7 @@ export function generateSdk(options: NgGenerateSdkSchema): Rule {
   return (tree, context) => {
     const isNx = isNxContext(tree);
     const workspaceConfig = getWorkspaceConfig(tree);
-    const defaultRoot = getPackagesBaseRootFolder(workspaceConfig, tree, 'library');
+    const defaultRoot = getPackagesBaseRootFolder(tree, context, workspaceConfig, 'library');
```

Comment:
Shouldn't you throw if `workspaceConfig` is undefined like in `packages/@o3r/core/schematics/library/index.ts`